python/ray/data/dataset.py

```diff
@@ -2591,6 +2574,9 @@ def make_generator():
                         f"but got a `{type(feature_columns).__name__}` instead."
                     )
 
+                if features.ndim == 1:
```

Comment:
Won't this fail if `features` is a dict or a tuple?

Reply:
Yeah, you're right. It would

Reply:
Fixed